htdocs/libraries/icms/view/block/Handler.php

```diff
@@ -219,7 +219,7 @@ public function getAllByGroupModule($groupid, $module_id = '0-0', $toponlyblock
 		$sql = "SELECT DISTINCT gperm_itemid FROM " . $this->db->prefix('group_permission')
 			. " WHERE gperm_name = 'block_read' AND gperm_modid = '1'";
 		if (is_array($groupid)) {
-			$gid = array_map(create_function('$a', '$r = "\'" . intval($a) . "\'"; return($r);'), $groupid);
+			$gid = array_map(function($a){$r = "\'" . intval($a) . "\'"; return($r);}, $groupid);
```

Comment:
I think it could be also written here easier: ``$gid = array_map('intval', $groupid);`` Extra quotes here don't do much.